tests/Lewee.Tests.Common/LogEntryExtensions.cs

```diff
@@ -6,7 +6,7 @@ namespace Lewee.Tests.Common;
 /// <summary>
 /// Extension methods for testing log entries
 /// </summary>
-public static class LogEntryExtensions
+internal static class LogEntryExtensions

```

Comment:
Please revert this back to `public` because `Lewee.Tests.Common` will be used by other test projects